### PR TITLE
RAS-1390|Add Registring Institiotion and Country into Trade Marks model

### PR DESCRIPTION
**Pull request by smaciejs**

### CODE REVIEW

File: src/ralph/trade_marks/models.py [hunk 1]
```diff
@@ -118,6 +142,11 @@ class TradeMark(Regionalizable, AdminAbsoluteUrlMixin, BaseObject):
         related_name='+',
         through='TradeMarksLinkedDomains',
     )
+    registering_institution = models.ForeignKey(
```
Comment (szymi-) [inline src/ralph/trade_marks/models.py]: Maybe `registrar` would be better? More concise and probably more natural. Google search for registering institution brings up no institution, only instructions on how to register an institution :)

File: src/ralph/trade_marks/models.py [hunk 2]
```diff
@@ -118,6 +142,11 @@ class TradeMark(Regionalizable, AdminAbsoluteUrlMixin, BaseObject):
...
+        TradeMarkRegisteringInstitution,
+        verbose_name=_('Registering Institution'),
```
Comment (szymi-) [inline src/ralph/trade_marks/models.py]: ditto

**Review by szymi- — CHANGES_REQUESTED** [review]
Also, there is still some reference to the old "region" field in views, which is causing tests to fail.

File: src/ralph/trade_marks/models.py (hunk 1 above)
Comment (smaciejs) [reply src/ralph/trade_marks/models.py]: Convinced me, I'll change it.

File: src/ralph/trade_marks/models.py (hunk 2 above)
Comment (smaciejs) [reply src/ralph/trade_marks/models.py]: Deleted

File: src/ralph/trade_marks/admin.py [hunk 3]
```diff
@@ -62,10 +68,26 @@ class TradeMarkAdmin(AttachmentsMixin, RalphAdmin):
             )
         })
     )
-    search_fields = ['name', 'id', ]
-    readonly_fields = ['image_tag']
+
+    class TradeMarksAdditionalCountyInline(RalphTabularInline):
```
Comment (Blejwi) [inline src/ralph/trade_marks/admin.py]: I guess there is a typo in `County`

**Review by Blejwi — CHANGES_REQUESTED** [review]

File: src/ralph/trade_marks/admin.py (hunk 3 above)
Comment (smaciejs) [reply src/ralph/trade_marks/admin.py]: Fixed

**Review by Blejwi — APPROVED** [review]

**Review by szymi- — APPROVED** [review]

**Pull request merged**